doc/CONTRIBUTING.md

```diff
@@ -337,6 +337,34 @@ for more configuration details. Everything within `python_config` is imported
 via `dictConfig` (in addition to some standard settings defined in
 `dataactcore.logging`.
 
+### Setup with Docker
```

Comment:
I don't see any instructions in here on how to set up the docker without spinning up a new DB, just pointing to an existing one.

Reply:
Bumping this comment

Reply:
I couldn't get my local postgres to work in the first place so I can't test how this would work. You should be able to change the config to point to your IP address but again I haven't been able to test it @alburde1 I have this line in the documentation if anyone wants to try it out "If you want to use postgres on your local machine, change the config to point to your host IP."

Reply:
I did some research and added more documentation of using host postgres with containers

Reply:
I was able to connect to my local database using docker.for.mac.host.internal since I'm running on docker 17.12.0.